### PR TITLE
add inline button with link to url

### PR DESCRIPTION
**Pull request by Sofbog**

### CODE REVIEW

**Comment by Sofbog** [comment]
[add inline button with link to url](https://github.com/glaring-brush/sticker_from_cutout_bot_telegram/pull/1/commits/ac2dc8757137e4a0c933d0098a5d80857cdb3bdf)

File: main.py [hunk 1]
```diff
@@ -23,20 +24,25 @@
 logger = logging.getLogger(__name__)
 
 TOKEN = os.getenv("TOKEN")
-WEBHOOK_URL = os.getenv("WEBHOOK_URL")
+# WEBHOOK_URL = os.getenv("WEBHOOK_URL")
```
Comment (glaring-brush) [inline main.py]: @Sofbog , could you please uncomment this line?

**Comment by glaring-brush** [comment]
@Sofbog , any updates on this?

**Comment by Sofbog** [comment]
> @Sofbog , any updates on this?

fixed the commented line

**Comment by glaring-brush** [comment]
Looks good 👍

**Pull request merged**